Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            string userFirstName = GetStringInput(message: "Enter your first name: ");
+            string userSurname = GetStringInput(message: "Enter your surname: ");
+            int userAge = GetIntInput(message: "Enter your age: ");
+            float userHeight = GetFloatInput(message: "Enter your height in cm: ");
+            float userWeight = GetFloatInput(message: "Enter your weight in kg: ");
+            float userBMI = CalculateBmi(userWeight, userHeight);
+            Console.WriteLine(userFirstName + " " + userSurname + " is " + userAge + " years old, his weight is " + userWeight + " kg and his height is " + userHeight + " cm.");
+            Console.WriteLine("BMI = " + userBMI);
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            if (weight <= 0 && height <=0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine("Weight cannot be equal or less than zero, but was " + weight);
+                Console.WriteLine("Height cannot be equal or less than zero, but was " + height);
```

Comment:
In **this** case and **only this case** the wording for height is different.
```suggestion
                Console.WriteLine("Height cannot be less than zero, but was " + height);
```